script/app-coverage-report.js

```diff
@@ -3,58 +3,84 @@
 const fs = require('fs');
 const path = require('path');
 const Table = require('cli-table');
+const {
+  getAppManifests,
+} = require('../src/site/stages/build/webpack/manifest-helpers');
 
 const printCoverage = coverageResults => {
   // Create a new table with headers
   const coverageTable = new Table({
-    head: ['Application', 'Lines', 'Functions', 'Statements', 'Branches'],
+    head: [
+      'Application (src/applications)',
+      'Lines',
+      'Functions',
+      'Statements',
+      'Branches',
+    ],
   });
 
   // Add each app coverage result to the table
-  Object.keys(coverageResults).forEach(key => {
-    if (key !== 'total') {
-      coverageTable.push({
-        [key]: [
-          `${coverageResults[key].lines.pct}%`,
-          `${coverageResults[key].functions.pct}%`,
-          `${coverageResults[key].statements.pct}%`,
-          `${coverageResults[key].branches.pct}%`,
-        ],
-      });
-    }
+  Object.values(coverageResults).forEach(cov => {
+    const appLocation = `${cov.path.substr(0, cov.path.lastIndexOf('/'))}`;
```

Comment:
This one doesn't have the trailing slash, so we don't need the template string.

```suggestion
    const appLocation = cov.path.substr(0, cov.path.lastIndexOf('/'));
```

Reply:
Whoops, didn't catch that, thanks!